src/stream_wrap.cc

```diff
@@ -224,6 +224,10 @@ void StreamWrap::WriteBuffer(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args.GetIsolate());
 
   StreamWrap* wrap = Unwrap<StreamWrap>(args.Holder());
+  if ( NULL == wrap) {
```

Comment:
Can you remove the space between `(` and `NULL` in all of your `if` statements. I think core normally writes as `wrap == NULL` as well.